frontend/src/container/ListOfDashboard/DashboardsList.tsx

```diff
@@ -459,17 +459,20 @@ function DashboardsList(): JSX.Element {
 									placement="left"
 									overlayClassName="title-toolip"
 								>
-									<Typography.Text data-testid={`dashboard-title-${index}`}>
-										<Link to={getLink()} className="title">
-											<img
-												src={dashboard?.image || Base64Icons[0]}
-												style={{ height: '14px', width: '14px' }}
-												alt="dashboard-image"
-												className="dashboard-icon"
-											/>
+									<Link to={getLink()} className="title-link">
+										<img
+											src={dashboard?.image || Base64Icons[0]}
+											style={{ height: '14px', width: '14px' }}
```

Comment:
@SagarRajput-7 avoid inline styling. Add rules to the class

Reply:
done